functions/src/modules/mewm/events/getEvent.ts

```diff
@@ -1,5 +1,35 @@
-import { ExpressFunction } from "../../../@types";
+import type { ExpressFunction } from "../../../@types";
+import { firebaseApp } from "../../../config/firebaseConfig";
+import { expressErrorHandlerFactory } from "../../../helpers";
+import { uasPermissionSwitch } from "../../../systems/uas";
 
-const replaceMe: ExpressFunction = (req, res, next) => {};
+const getEvent: ExpressFunction = (req, res, next) => {
+  uasPermissionSwitch({
+    organizer: { accepted: execute },
+    sponsor: { accepted: execute },
+    mentor: { accepted: execute },
+    volunteer: { accepted: execute },
+    hacker: { accepted: execute },
+  })(req, res, next);
+};
 
-export default replaceMe;
+const execute: ExpressFunction = (req, res, next) => {
+  const errorHandler = expressErrorHandlerFactory(req, res, next);
+  firebaseApp
+    .firestore()
+    .collection("miniEvents")
+    .doc(req.params.eventId)
+    .get()
+    .then((document) => {
+      const data = document.data() as METMMiniEvent | undefined;
```

Comment:
Heads up! It's the "Mini Events and Workshops Module" (MEWM). Also, I think that these are just referred to as "Events" and not "Mini Events" as they can also include things like deadlines or meals.
```suggestion
      const data = document.data() as MEWMEvent | undefined;
```